cmd/aro/operator.go

```diff
@@ -100,7 +106,7 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 		}
 		if err = (workaround.NewReconciler(
 			log.WithField("controller", controllers.WorkaroundControllerName),
-			kubernetescli, configcli, mcocli, arocli, restConfig)).SetupWithManager(mgr); err != nil {
+			kubernetescli, env, configcli, mcocli, securitycli, arocli, restConfig)).SetupWithManager(mgr); err != nil {
```

Comment:
If restConfig is being passed `securityCli` can be implementation details under the hood?

Reply:
Or this is somehow different?

Reply:
@mjudeikis Routefix needs the securitycli to grab the "privileged" scc and create one for the Routefix dameonset.  For the workaround controller to Ensure() on the workaround Routefix, the securitycli is necessary.

If we want to return the scc and not get => create/update the scc, I can remove the securitycli and create a function that returns the scc object as it would if it was created from the privileged scc.

Reply:
my question is why you pass int `securitycli` as argument if restConfig goes into arguments too.
When this happens:
```
securitycli, err := securityclient.NewForConfig(restConfig)
	if err != nil {
		return err
	}
```
Or is this for tests?